checkout.html

```diff
@@ -0,0 +1,177 @@
+
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>Adopt-A-Dog - Checkout</title>
+        <link rel="stylesheet" type="text/css" href="styles/stylesheet.css">
+        <link rel="stylesheet"
+          href="https://fonts.googleapis.com/css?family=Work+Sans|Neuton">
+    </head>
+    <body>
+        <header>
+            <div class="header-logo-checkout-overlay">
+                <div class="header-logo-wrapper">
+                    <img src="images/adoptadog-logo.svg" alt="adopt-a-dog logo">
+                    <h1 class="header-logo-heading">Seattle’s premiere dog adoption agency</h1>
+                </div>
+                <div class="header-checkout-wrapper">
+                    <img class="header-checkout-icon" src="images/cart-icon.svg" alt="checkout cart icon">
+                    <p class="header-checkout-cart">$1234.56</p>
+                    <a href="checkout.html" class="header-checkout-link">Checkout</a>
+                </div>
+            </div>
+            
+            <nav class="header-nav">
+                <a href="index.html">Home</a>
+                <span>/</span>
+                <a href="dogs.html">Dogs</a>
+                <span>/</span>
+                <a href="blog.html">Blog</a>
+            </nav>
+        </header>
+    <main>
+        <div class="main-heading"><h1>Checkout</h1></div>
+        <form onsubmit="onSubmit();formValidation()">
```

Comment:
Might be better practice to combine both into one function. Also your page reloads before console can be viewed. you can prevent this by returning false in your onSubmit function. 

Combine both into one function and do <form onsubmit="return onSubmit()">

Then at the end of your onSubmit function, return false.